README.md

```diff
@@ -1,44 +1,14 @@
-
-## About This Project
-
-The concept for this project originated around 2014 while developing a custom UNIX-like proprietary RTOS for IoT applications. The concept was reinforced year after year, observing that several companies had developed their own serialization and deserialization protocols which often lacked robustness of other established and standardized protocols. Moreover, the stacks would require a rather large effort to rework whenever the hardware platform or physical transport changed.
-
-After becoming familiar with the concept of [automatic programming](https://en.wikipedia.org/wiki/Automatic_programming), looking at older solutions such as [ASN.1](https://en.wikipedia.org/wiki/ASN.1), [SunRPC](https://en.wikipedia.org/wiki/Sun_RPC) as well as newer solutions such as [Protocol Buffers](https://en.wikipedia.org/wiki/Protocol_Buffers) and [gRPC](https://grpc.io/), Thrift was beginning to seem very appealing.
-
-Many years later, in March of 2022, during a Hackathon at [Meta](https://meta.com/), a group of [Meta Engineers](https://engineering.fb.com) came together because they knew there was a better way to communicate with firmware using Thrift. Zephyr was a clear winner in the RTOS market due to
-* support for standards such as [POSIX](https://en.wikipedia.org/wiki/POSIX) and [BSD Sockets](https://en.wikipedia.org/wiki/Berkeley_sockets)
-* support for a huge number of standardized physical layer protocols
-* making the right choice to use Kconfig for software configuration and DevcieTree for hardware configuration
-* and lastly, because the Zephyr community was an incredibly welcoming and positive space
-
-The Hackathon became a success in just a few days!
-
-# Google Summer of Code, 2022
-
-As announced on the [Linux Foundation Wiki](https://wiki.linuxfoundation.org/gsoc/2022-gsoc-zephyr), this project was selected for the [Google Summer of Code](https://summerofcode.withgoogle.com/) under the umbrella of the [Linux Foundation](https://linuxfoundation.org). View the accepted proposal [here](https://summerofcode.withgoogle.com/proposals/details/CROr49Ia).
-
-The progress of this project is tracked in [the project board](https://github.com/orgs/zephyrproject-rtos/projects/11) and visualized with [ganttlab](https://www.ganttlab.com/). Check our weekly update:
+## Build Status
 
-![Gantt chart](doc/gantt/latest.png)
+[![Build](https://github.com/cfriedt/thrift-for-zephyr/actions/workflows/build.yml/badge.svg)](https://github.com/cfriedt/thrift-for-zephyr/actions)
```

Comment:
might need to change this away from `cfriedt/thrift-for-zephyr` to `zephyrproject-rtos/gsoc-2022-thrift`